[2024-08-06T21:41:39.287+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: load_parquet_to_bq_dag.dbt_run manual__2024-08-06T21:34:27.954619+00:00 [queued]>
[2024-08-06T21:41:39.291+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: load_parquet_to_bq_dag.dbt_run manual__2024-08-06T21:34:27.954619+00:00 [queued]>
[2024-08-06T21:41:39.291+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-08-06T21:41:39.292+0000] {taskinstance.py:1280} INFO - Starting attempt 2 of 2
[2024-08-06T21:41:39.292+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-08-06T21:41:39.298+0000] {taskinstance.py:1300} INFO - Executing <Task(DbtRunOperator): dbt_run> on 2024-08-06 21:34:27.954619+00:00
[2024-08-06T21:41:39.300+0000] {standard_task_runner.py:55} INFO - Started process 52 to run task
[2024-08-06T21:41:39.302+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'load_parquet_to_bq_dag', 'dbt_run', 'manual__2024-08-06T21:34:27.954619+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/load_dbt_dag.py', '--cfg-path', '/tmp/tmp5p0v2_zp']
[2024-08-06T21:41:39.302+0000] {standard_task_runner.py:83} INFO - Job 57: Subtask dbt_run
[2024-08-06T21:41:39.328+0000] {task_command.py:388} INFO - Running <TaskInstance: load_parquet_to_bq_dag.dbt_run manual__2024-08-06T21:34:27.954619+00:00 [running]> on host cf0df9e834a7
[2024-08-06T21:41:39.352+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=miguel
AIRFLOW_CTX_DAG_ID=load_parquet_to_bq_dag
AIRFLOW_CTX_TASK_ID=dbt_run
AIRFLOW_CTX_EXECUTION_DATE=2024-08-06T21:34:27.954619+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-06T21:34:27.954619+00:00
[2024-08-06T21:41:39.352+0000] {dbt_hook.py:117} INFO - dbt run --profiles-dir /opt/dbt_project --models load_parquet_to_test
[2024-08-06T21:41:39.356+0000] {dbt_hook.py:126} INFO - Output:
[2024-08-06T21:41:40.479+0000] {dbt_hook.py:130} INFO - [0m21:41:40  Running with dbt=1.8.4
[2024-08-06T21:41:42.140+0000] {dbt_hook.py:130} INFO - [0m21:41:42  Registered adapter: bigquery=1.8.2
[2024-08-06T21:41:42.150+0000] {dbt_hook.py:130} INFO - [0m21:41:42  Encountered an error:
[2024-08-06T21:41:42.151+0000] {dbt_hook.py:130} INFO - Compilation Error
[2024-08-06T21:41:42.152+0000] {dbt_hook.py:130} INFO -   dbt found 3 package(s) specified in packages.yml, but only 0 package(s) installed in dbt_packages. Run "dbt deps" to install package dependencies.
[2024-08-06T21:41:43.244+0000] {dbt_hook.py:132} INFO - Command exited with return code 2
[2024-08-06T21:41:43.252+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow_dbt/operators/dbt_operator.py", line 98, in execute
    self.create_hook().run_cli('run')
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow_dbt/hooks/dbt_hook.py", line 138, in run_cli
    raise AirflowException("dbt command failed")
airflow.exceptions.AirflowException: dbt command failed
[2024-08-06T21:41:43.255+0000] {taskinstance.py:1318} INFO - Marking task as FAILED. dag_id=load_parquet_to_bq_dag, task_id=dbt_run, execution_date=20240806T213427, start_date=20240806T214139, end_date=20240806T214143
[2024-08-06T21:41:43.263+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 57 for task dbt_run (dbt command failed; 52)
[2024-08-06T21:41:43.299+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-08-06T21:41:43.309+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
